src/main/java/se/lth/base/server/data/User.java

```diff
@@ -1,31 +1,96 @@
 package se.lth.base.server.data;
 
 import java.security.Principal;
+import java.sql.Timestamp;
 
 public class User implements Principal {
 
-    public static User NONE = new User(0, Role.NONE, "-");
+    public static User NONE = new User(0, Role.NONE, "-", "-", "-", "0", "-", 0, Timestamp.valueOf("2018-01-01 00:00:00"), false, 0, 0, "-");
 
-    private final int id;
+    private final int user_id;
     private final Role role;
     private final String username;
+    private final String first_name;
+    private final String last_name;
+    private final String phone_number;
+    private final String email;
+    private final int gender;
+    private final Timestamp date_of_birth;
+    private final Boolean driving_licence;
+    private final int rating_total_score;
+    private final int number_of_ratings;
+    private final String warning;
```

Comment:
Shouldn't this be an integer, i.e. the number of warnings?

Reply:
som jag förstått så är warning själva meddelandet.

Reply:
As I see it, this should probably be a number right? What should we do with the message? Look at the mockups

<img width="518" alt="screen shot 2018-10-02 at 19 32 29" src="https://user-images.githubusercontent.com/22319122/46365846-e7f90800-c679-11e8-8d5a-0f194149935b.png">